sdmetrics/column_pairs/statistical/inter_row_msas.py

```diff
@@ -76,8 +76,17 @@ def compute(real_data, synthetic_data, n_rows_diff=1, apply_log=False):
         synthetic_keys, synthetic_values = synthetic_data
 
         if apply_log:
-            real_values = np.log(real_values)
-            synthetic_values = np.log(synthetic_values)
+            num_invalid = sum(x <= 0 for x in pd.concat((real_values, synthetic_values)))
+            if num_invalid:
+                warnings.warn(
+                    f'There are {num_invalid} non-positive values in your data, which cannot be '
+                    "used with log. Consider changing 'apply_log' to False for a better result."
+                )
+            with warnings.catch_warnings():
+                warnings.filterwarnings('ignore', message='divide by zero encountered in log')
+                warnings.filterwarnings('ignore', message='invalid value encountered in log')
```

Comment:
if we just used 'encountered in log' , wouldn't it be enough ? So have just one line.